src/styles/components/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
+
+    @extend %bgc-image;
+
+    background-image: url('../images/star.svg');
```

Comment:
The path to the images seems to be hardcoded. This can lead to problems if the directory structure changes. Consider storing the path in a variable, so you can easily change it in one place if needed.